TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -534,18 +543,15 @@ public static class PriorityRoad {
             }
             ref NetSegment seg = ref segmentId.ToSegment();
             ref NetNode node = ref nodeId.ToNode();
-            bool startNode = (bool)ExtSegmentManager.Instance.IsStartNode(segmentId, nodeId);
-            ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
+            bool startNode = seg.IsStartnode(nodeId);
 
             //list of outgoing lanes from current segment to current node.
-            IList<LanePos> laneList =
-                extSegmentManager.GetSortedLanes(
-                    segmentId,
-                    ref seg,
-                    startNode,
-                    LaneArrowManager.LANE_TYPES,
-                    LaneArrowManager.VEHICLE_TYPES,
-                    true);
+            var laneList = seg.GetSortedLanes(
+                startNode,
+                LaneArrowManager.LANE_TYPES,
+                LaneArrowManager.VEHICLE_TYPES,
+                reverse: true); // should this be `!lht`?
+
```

Comment:
I thought `GetSortedLanes()`considers LHT. Is that right?

Reply:
> I thought `GetSortedLanes()` considers LHT. Is that right?

No, it doesn't check that specific property either directly or via the shortcut.

Do we even need the `reverse: true` here though - does order of returned lanes matter in this context? If so, should it depend on LHT or should it always be reversed?

Reply:
I think I have handled it like this             
```
int sideLaneIndex = lht ? srcLaneCount - 1 : 0;
```
but I think it would work if you simply play with reverse: !lht too. 
I think you can work out the math.
test it in a junction explainedby the comment at line 584:
```
          /* in case there are multiple minor roads attached to the priority road at the same side
             * and the main road is straigh, then add a turn arrow into the other minor roads.
             */
```
AN mod allows instant change of LHT/RHT in game.

Reply:
I've left it as original functionality just to keep this PR simple.